trafilatura/metadata.py

```diff
@@ -194,10 +211,12 @@ def examine_meta(tree):
                 tags.append(content_attr)
         elif 'itemprop' in elem.attrib:
             if elem.get('itemprop') == 'author':
-                if author is None:
-                    author = content_attr
-                elif author not in content_attr and not content_attr.startswith('http'):
-                    author = author + '; ' + content_attr
+                authors = normalize_authors(content_attr).split(';')
```

Comment:
It also didn't occur to me that the same code is repeated thrice. It could be moved to `normalize_authors` or at least to a separate function?